### PR TITLE
#66 - Implement button with the linear gradient color 

### LINKED ISSUE #66: Implement button with the linear gradient color
## Description:

Add the gradient color to the palette and implement the gradient button.

Gradient color: linear-gradient(89.58deg, #44BCF0 -19.85%, #818CF8 54.07%, #A099FF 120.75%)

### PR DESCRIPTION
**Pull request by Koushith**
## Description

- [x] Adds Linear gradient color.

## Screenshot
image

## Platform tested
- [x] Large
- [x] Tablet
- [x] Mobile

### CODE REVIEW

File: src/themes/palette.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export const palette = {
   greyLighter: '#8E90A6', //for placeholders and secondary button, border
   grayTransparent: 'rgba(98, 108, 116, 0.4)', //model backdrop
   shadow: ' 0px 16px 32px rgba(221, 230, 237, 0.4)',
+  linearGradient: 'linear-gradient(89.58deg, #44BCF0 -19.85%, #818CF8 54.07%, #A099FF 120.75%)',
```
Comment (koshikraj) [inline src/themes/palette.ts]: Name this gradient color accordingly. Maybe `blueIndigoGradient`

Comment (Koushith) [reply src/themes/palette.ts]: yes, renamed in the latest commit.

**Review by koshikraj — APPROVED** [review]

File: src/components/primitive/button/button.component.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const Button: React.FunctionComponent<ButtonComponentProps> = (props: But
    * custom styling goes here.
    */
   const StyledButtonComponent = styled(StyledButton)<ButtonComponentProps>`
-    background-color: ${({ color, theme: { colors } }) => color && colors[color]} !important;
+    background: ${({ color, theme: { colors } }) => color && colors[color]} !important;
```
Comment (dsouzaedison) [inline src/components/primitive/button/button.component.tsx:34]: What happens if colors[color] is undefined or if color is undefined? Will it print `background: undefined !important;` ?

**Review by dsouzaedison — APPROVED** [review]

File: src/components/primitive/button/button.component.tsx (hunk 2 above)
Comment (Koushith) [reply src/components/primitive/button/button.component.tsx:34]: That's the option props. all the custom styling goes here. if the color is undefined, the Primary color will be rendered. i.e coming from button.component.styles.ts file.

**Pull request merged**